index.html

```diff
@@ -195,6 +195,32 @@ <h6 class="about-title">
       </div>
     </div>
   </section>
+  <footer>
+    <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+    <form action="">
+      <input type="text" maxlength="30" placeholder="Full name" required name="full-name">
+      <input type="email" placeholder="Email" required name="email">
+      <textarea name="" id="" maxlength="500" placeholder="Message" name="message" required></textarea>
+    </form>
```

Comment:
### Missing Button :record_button:
![Screenshot from 2022-09-28 17-57-39 (copy)](https://user-images.githubusercontent.com/93039359/192786744-60604a34-da4d-4dd1-a5b3-8b72546fd4ee.png)


- [ ] I was thinking that how i'm gonna submit my form then i saw that there is not submit button :smile: No problem you can add it like below :arrow_down:
`<button type='submit'>Get in touch<button>`

- [ ] Kindly, add the `hover` effect on the button as well in your css file to make your button look familiar with other form buttons :+1: